tests/core/test_astro_cartography.py

```
@@ -75,16 +75,51 @@ def test_step_affects_detail(self, subject):
         fine = AstroCartographyFactory.compute(subject, step=2, planets=["Sun"])
 
         # MC lines should have more points with finer step
-        coarse_mc = [l for l in coarse if l.line_type == "MC"][0]
-        fine_mc = [l for l in fine if l.line_type == "MC"][0]
+        coarse_mc = [line for line in coarse if line.line_type == "MC"][0]
```

Ruff (0.15.15)

[warning] Prefer `next(line for line in coarse if line.line_type == "MC")` over single element slice

Replace with `next(line for line in coarse if line.line_type == "MC")`

(RUF015)